WMFComponents/Sources/WMFComponents/Components/Activity Tab/WMFActivityTabViewModel.swift

```diff
@@ -183,4 +56,77 @@ public class WMFActivityTabViewModel: ObservableObject {
             self.loggedOutSecondaryCTA = loggedOutSecondaryCTA
         }
     }
+
+    public let localizedStrings: LocalizedStrings
+    @Published public var isLoggedIn: Bool
+
+    @Published public var articlesReadViewModel: ArticlesReadViewModel {
+        didSet { bindChildPublishers() }
+    }
+    @Published public var articlesSavedViewModel: ArticlesSavedViewModel {
+        didSet { bindChildPublishers() }
+    }
```

Comment:
Are we sure this child publishers complication is needed? I would have thought consistent use of `@Published` just works.